### PR TITLE
Feat/add form toggle button

### PR DESCRIPTION
**Pull request by alexanderleegs**
This PR adds a new class for form fields which have only 2 options, the FormFieldToggle button. This PR is meant to resolve https://github.com/isomerpages/isomercms-frontend/issues/307.

The now unused FormFieldDropdown class has also been removed in this PR.

Preview of FormFieldToggle buttons:
Screenshot 2020-12-30 at 3 01 43 PM

Screenshot 2020-12-30 at 3 01 07 PM

### CODE REVIEW

File: src/components/FormFieldToggle.jsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+const creatableSelectHandler = (callback, dropdownEvent, id) => {
+    callback({
+        target: {
+            id,
+            value: dropdownEvent,
+        },
+    })
+}
+
+const FormFieldDropdown = ({
```
Comment (gweiying) [inline src/components/FormFieldToggle.jsx:15]: should be `FormFieldToggle`

File: src/components/FormFieldToggle.jsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                thumbIcon={<span className={`bx bx-xs ${value ? `${elementStyles.formToggleThumbIconEnabled} bx-check` : `${elementStyles.formToggleThumbIconDisabled} bx-x`}`}/>}
+                value={value}
+                onToggle={(value) => creatableSelectHandler(onFieldChange, !value, id)}
+            />
+        </div>
+      </div>
+    </>
+);
+  
+export default FormFieldDropdown;
```
Comment (gweiying) [inline src/components/FormFieldToggle.jsx:40]: should be `FormFieldToggle`

File: src/components/FormFieldToggle.jsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  
+FormFieldDropdown.propTypes = {
```
Comment (gweiying) [inline src/components/FormFieldToggle.jsx:42]: should be `FormFieldToggle`

**Review by gweiying — CHANGES_REQUESTED** [review]
Small typo in `src/components/FormFieldToggle.jsx`, otherwise the functionality looks good. 

Small suggestion, maybe we can consider changing the alignment or thickness of the toggle, otherwise it looks slightly off with respect to the other fields? See the image below:
Screenshot 2020-12-30 at 11 14 59 AM

File: src/components/FormFieldToggle.jsx (hunk 2 above)
Comment (alexanderleegs) [reply src/components/FormFieldToggle.jsx]: Good catch, resolved in https://github.com/isomerpages/isomercms-frontend/pull/321/commits/257e58a58af8a4f1e2a868f2ce5fdfa23a2b453c

File: src/components/FormFieldToggle.jsx (hunk 3 above)
Comment (alexanderleegs) [reply src/components/FormFieldToggle.jsx]: https://github.com/isomerpages/isomercms-frontend/pull/321/commits/257e58a58af8a4f1e2a868f2ce5fdfa23a2b453c

**Comment by alexanderleegs** [comment]
> Small typo in `src/components/FormFieldToggle.jsx`, otherwise the functionality looks good.
> 
> Small suggestion, maybe we can consider changing the alignment or thickness of the toggle, otherwise it looks slightly misaligned with respect to the other fields? See the Footer toggle in the image below:
> Screenshot 2020-12-30 at 11 14 59 AM

That's a good point, I've increased the padding in https://github.com/isomerpages/isomercms-frontend/pull/321/commits/56d94b032013772da3c0682f1971f3a37ab5e41e.

I've also shifted the reach toggle button to the bottom of the footer settings so it doesn't look as jarring.
Screenshot 2020-12-31 at 10 29 51 AM

**Review by gweiying — APPROVED** [review]
lgtm

**Pull request merged**